app/src/main/res/layout/fragment_blinds_settings.xml

```diff
@@ -11,4 +12,32 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content" />
 
+
+    <!-- add blinds button -->
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/add_btn"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentBottom="true"
+        android:layout_gravity="bottom|right"
```

Comment:
```suggestion
        android:layout_gravity="bottom|end"
```
Android studio gives a warning here about using end instead of right for left-to-right locales. Please do not ignore warnings.